GVFS/GVFS.FunctionalTests/FileSystemRunners/SystemIORunner.cs

```diff
@@ -19,6 +20,17 @@ public override string MoveFile(string sourcePath, string targetPath)
         {
             File.Move(sourcePath, targetPath);
             return string.Empty;
+        }
+
+        public override void CreateFileWithoutClose(string path)
+        {
+            File.Create(path);
+        }
+
+        public override void OpenFileAndWriteWithoutClose(string path, string content)
+        {
+            StreamWriter file = new StreamWriter(path);
+            file.Write(content);

```

Comment:
Similarly to the `CreateFileWithoutClose` case, I'm not clear on how this inhibits a `close()` syscall; also, is the `StreamWriter` buffered by default? If the buffers aren't flushed, the write won't hit disk, so you wouldn't expect `git status` to pick them up?

Reply:
@pmj I validated the behavior by turning on the test and running a few times.  When I address the related bug I'll dig more.  If AuthFlush isn't true it's probably worth adding a .Flush for good measure.  Thank you!

Reply:
Yeah, I couldn't work out from the docs if `AutoFlush` was true by default.